src/sentry/api/bases/organization.py

```diff
@@ -366,6 +367,17 @@ def convert_args(self, request: Request, organization_slug, *args, **kwargs):
         kwargs["organization"] = organization
         return (args, kwargs)
 
+    def get_use_case_id(self, request: Request):
+        """
+        Extracts useCase from request and validate it agains UseCaseKey enum type
+        if request param is omitted fallback to UseCaseKey.RELEATH_HEALTH
+        if request param has wrong value just raise an ParseError.
+        """
+        try:
+            return UseCaseKey.from_str(request.GET.get("useCase", "release-health"))
+        except ValueError:
+            raise ParseError(detail="Invalid useCase parameter.")
```

Comment:
Is there a list of valid choices that we could tell the user to choose from?

Reply:
yeah one moment.

Reply:
@markstory PTAL a230df69c2